docs/enterprise-api.md

```diff
@@ -835,6 +848,17 @@ message ExecuteWorkflowRequest {
   // overrides will take precedence. Otherwise all env vars set in
   // buildbuddy.yaml will still apply.
   map<string, string> env = 7;
+
+  message ReportingMetadata {
+    string branch = 1;
+    string commit_sha = 2;
+  }
+  // Github statuses should be reported with these fields.
```

Comment:
I think it's worth clarifying what kinds of reporting we're referring to - there is info reported both to BB (build metadata) and GH (commit status)

Reply:
Fixed in the other PR